tasks/main.yml

```diff
@@ -66,6 +66,25 @@
       check_mode: "no"
       register: host_info
 
+    - name: Get Host volumes
+      no_log: false
+      uri:
+        url: "{{ ovirt_auth.url }}/clusters/{{ cluster_info.ovirt_clusters[0].id }}/glustervolumes"
+        method: GET
+        body_format: json
+        validate_certs: false
+        return_content: yes
+        headers:
+          Authorization: "Bearer {{ ovirt_auth.token }}"
+          Accept: "application/json"
+      register: host_volumes
```

Comment:
You should use ovrit_cluster_info module. Update taks `Get cluster info` 
```yml
    - name: Get cluster info
      ovirt_cluster_info:
        auth: "{{ ovirt_auth }}"
        fetch_nested: True
        nested_attributes: name_of_atr
        pattern: "name={{ cluster_name }}"
      check_mode: "no"
      register: cluster_info
```

Reply:
I checked this, It will fetch only IDs of nested entity. i require volume names. Ref: https://docs.ansible.com/ansible/latest/modules/ovirt_cluster_info_module.html#parameters

Reply:
You need to use `nested_attributes: name_of_atr` to fetch attributes which you need,
nested_attributes is list so you can put multiple attributes in it.